utils/webpack/webpack.config.js

```diff
@@ -31,6 +32,11 @@ const t = i18n(__filename);
 const devtool = isDev ? sourceMap : (process.env.SOURCE_MAPS || false);
 const fileSuffix = devtool ? '.sm' : '';
 
+// This is a workaround for the issue with the md4 hash algorithm after upgrading to node:18
+// Error: error:0308010C:digital envelope routines::unsupported
+const crypto_orig_createHash = crypto.createHash;
```

Comment:
this workaround might not work anymore with the latest webpack 4 version 4.47.0. this version seems to contain a fix for the issue that you tried to handle with this workaround. 

https://github.com/webpack/webpack/releases/tag/v4.47.0

within the pull request to fix the node 18 issue, they removed the crypto module from some files:

https://github.com/webpack/webpack/pull/17628/files

so your workaround might not work anymore / is not necessary anymore.

within the redux update branch webpack was updated to this version. it's already merged to `develop7`, so you can check the issue still happens inside.